### PR TITLE
Update jax 0.3.21

### PR DESCRIPTION
**Pull request by ppwwyyxx**

### CODE REVIEW

File: docsets/jax/docset.json [hunk 1]
```diff
@@ -1,10 +1,28 @@
 {
     "name": "jax",
-    "version": "0.1.67",
-    "archive": "jax.tgz",
+    "version": "0.3.21",
+    "archive": "Jax-0.3.21.tgz",
```
Comment (Kapeli) [inline docsets/jax/docset.json:4]: Please don't include a version in the archive name.

File: docsets/jax/docset.json [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
     "author": {
-        "name": "Niru Maheswaranathan",
-        "link": "https://twitter.com/niru_m"
+        "name": "Yuxin Wu",
+        "link": "https://github.com/ppwwyyxx"
     },
-    "aliases": ["jaxlib"]
+    "aliases": ["jaxlib", "flax"],
+    "specific_version": [
```
Comment (Kapeli) [inline docsets/jax/docset.json:10]: This key is wrong. It should be `specific_versions`.

**Comment by ppwwyyxx** [comment]
Hi I resolved the comments. Please take a look again, thanks!

**Comment by Kapeli** [comment]
Thank you! ❤️

**Pull request merged**